poi/src/test/java/org/apache/poi/hssf/extractor/TestExcelExtractor.java

```diff
@@ -347,7 +347,7 @@ void testPassword() throws IOException {
 
     @Test
     void testNullPointerException() throws IOException {
-        try (ExcelExtractor extractor = createExtractor("ar.org.apsme.www_Form%20Inscripcion%20Curso%20NO%20Socios.xls")) {
+        try (ExcelExtractor extractor = createExtractor("ar.org.apsme.www_Form%20Inscripción%20Curso%20NO%20Socios.xls")) {
```

Comment:
Err, this wasn't the correction I expected to get. I was expecting to leave `Inscription`...

I'm curious what this is doing...

Reply:
Oops, finally found the file in the repository. Ok, I'm just dropping this change.